app/src/main/java/com/codesoom/assignment/application/CommandProductService.java

```diff
@@ -0,0 +1,56 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.ProductNotFoundException;
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+public class CommandProductService {
+
+    private final ProductRepository productRepository;
+
+    public CommandProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    public Product createProduct(Product requestProduct) {
+        return productRepository.save(requestProduct);
+    }
+
+    public Product updateProduct(Long id, Product requestProduct) {
+        Product product = productRepository.findById(id).orElseThrow(() -> new ProductNotFoundException(id));
+
+        String name = requestProduct.getName();
+        Integer price = requestProduct.getPrice();
+        String maker = requestProduct.getMaker();
+        String imageUrl = requestProduct.getImageUrl();
+
+        if (name != null) {
+            product.changeNameTo(name);
+        }
+        if (price != null) {
+            product.changePriceTo(price);
+        }
+        if (maker != null) {
+            product.changeMakerTo(maker);
+        }
+        if (imageUrl != null) {
+            product.changeImageUrlTo(imageUrl);
+        }
```

Comment:
```suggestion
        if (name != null) {
            product.changeNameTo(requestProduct.getName());
        }
        if (price != null) {
            product.changePriceTo(requestProduct.getPrice());
        }
        if (maker != null) {
            product.changeMakerTo(requestProduct.getMaker());
        }
        if (imageUrl != null) {
            product.changeImageUrlTo(requestProduct.getImageUrl());
        }
```
변수명을 없애도 가독성은 유지되는 것 같은습니다!
어떻게 생각하시는지 궁금합니다!

Reply:
```
        if (requestProduct.getName() != null) {
            product.changeNameTo(requestProduct.getName());
        }
```
영환님 의견대로 변수를 없애면 if 문 안에서 사용할 수가 없어서 더 복잡해 보일 수 있을 것 같습니다!